multiwallet.py

```diff
@@ -246,6 +246,35 @@ def _get_bip39_firstwords():
         return fw
 
 
+#####################################################################
+# Dice rolls
+#####################################################################
+
+
+def _get_num_words():
+    num_words = _get_int(
+        prompt="How many mnemonic words should be generated?",
+        default=24,
+        minimum=12,
+        maximum=24,
+    )
+    if num_words not in (12, 15, 18, 21, 24):
+        raise ValueError(
+            "Provided number of mnemonic words must be 12, 15, 18, 21, or 24 words"
+        )
+    return num_words
+
+
+def _get_dice_values():
+    dice_vals = input(
+        blue_fg(
+            "Enter the numbers from the 6-sided dice rolls"
+            '\n(e.g., "14625363...", >= 100 values recommended): '
+        )
+    ).strip()
+    return dice_vals
```

Comment:
This needs to handle bad entries (like someone entering a letter, space, a number not 1-6, etc) gracefully